aircast/rootfs/usr/bin/shairport-play-handler.py

```
@@ -0,0 +1,230 @@
+            'pipe_path': pipe_path,
+            'port': port,
+            'ffmpeg_pid': handler.ffmpeg_process.pid if handler.ffmpeg_process else None,
+            'http_pid': handler.http_server_process.pid if handler.http_server_process else None
+        }, f)
+
+def handle_stop(entity_id: str):
+    """Handle playback stop"""
+    print(f"Playback stopping for {entity_id}", file=sys.stderr)
+    
+    # Read state file
+    state_file = f"/tmp/shairport_state_{entity_id.replace('.', '_')}.json"
```

Ruff (0.14.7)

Probable insecure usage of temporary file or directory: "/tmp/shairport_state_"

(S108)